### PR TITLE
update readme fetch integration docs

### PR DESCRIPTION
**Pull request by jensendarren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ contract PriceContract is UsingFetch {
   constructor(address payable _fetchAddress) UsingFetch(_fetchAddress) {}
 
   function getPrice() public view returns (uint256) {
-      bytes32 _plsQueryId = keccak256(abi.encode("SpotPrice", abi.encode("pls", "usd")));
```
Comment (KolomboPulse) [inline README.md:20]: pls

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ contract PriceContract is UsingFetch {
...
+      bytes32 _plsQueryId = keccak256(abi.encode("SpotPrice", abi.encode("llpls", "usd")));
```
Comment (KolomboPulse) [inline README.md:20]: llpls

File: README.md [hunk 3]
```diff
@@ -31,9 +31,8 @@ contract PriceContract is UsingFetch {
 
 Find Fetch contract addresses:
 
-mainnet: 0x6f390b99201bb43A05757019efe9C99651e04584
```
Comment (KolomboPulse) [inline README.md:34]: mainnet: 0x6f390b99201bb43A05757019efe9C99651e04584

File: README.md [hunk 4]
```diff
@@ -31,9 +31,8 @@ contract PriceContract is UsingFetch {
...
-testnet: 0x252eC80dEa7F3eD0CC57e0f1112d6f56Ae9523fb
```
Comment (KolomboPulse) [inline README.md:35]: testnet: 0x252eC80dEa7F3eD0CC57e0f1112d6f56Ae9523fb

File: README.md [hunk 5]
```diff
@@ -31,9 +31,8 @@ contract PriceContract is UsingFetch {
...
-
+* mainnet: `0x23716E2dCC327429253335E4646dEbaD1Ac0E57c`
```
Comment (KolomboPulse) [inline README.md:34]: * mainnet: `0x23716E2dCC327429253335E4646dEbaD1Ac0E57c`

File: README.md [hunk 6]
```diff
@@ -31,9 +31,8 @@ contract PriceContract is UsingFetch {
...
+* testnet: `0x252eC80dEa7F3eD0CC57e0f1112d6f56Ae9523fb`
```
Comment (KolomboPulse) [inline README.md:35]: * testnet: `0x252eC80dEa7F3eD0CC57e0f1112d6f56Ae9523fb`

File: README.md [hunk 7]
```diff
@@ -196,9 +195,6 @@ npm i
 npm test
 ```
 
-# Implementing using Fetch
```
Comment (KolomboPulse) [inline README.md:199]: # Implementing using Fetch

File: README.md [hunk 8]
```diff
@@ -196,9 +195,6 @@ npm i
...
-See our documentation for implementing usingFetch [here.](https://docs.fetchoracle.com/fetch/getting-data/introduction)
```
Comment (KolomboPulse) [inline README.md:200]: See our documentation for implementing usingFetch [here.](https://docs.fetchoracle.com/fetch/getting-data/introduction)

**Review by KolomboPulse — APPROVED** [review]

**Comment by darren-ll** [comment]
Hi @KolomboPulse 👋 - thank you for your earlier comments on this PR. Can you help to share a little more context please so that I may help you?

File: README.md (hunk 1 above)
Comment (KolomboPulse) [inline README.md:20]: pls

File: README.md (hunk 8 above)
Comment (KolomboPulse) [inline README.md:200]: See our documentation for implementing usingFetch [here.](https://docs.fetchoracle.com/fetch/getting-data/introduction)

**Review by KolomboPulse — COMMENTED** [review]
update readme fetch integration docs #10

**Review by KolomboPulse — COMMENTED** [review]
update readme fetch integration docs #10

File: README.md (hunk 7 above)
Comment (KolomboPulse) [inline README.md:199]: # Implementing using Fetch

**Review by KolomboPulse — CHANGES_REQUESTED** [review]

**Pull request merged**